### ISSUE TITLE
Typo: "hostname of FQDN"

### ISSUE THREAD

**Issue by jihoon-seo** [kubernetes/website#31862]
**This is a Bug Report**

**Problem:**
In [here](https://kubernetes.io/docs/tasks/configure-pod-container/configure-gmsa/#:~:text=network%20shares%20usinig-,hostname%20of%20FQDN,-If%20you%20are),
the section title contains "hostname of FQDN",
but considering the context,
it should be "hostname or FQDN".

(You can see that in the paragraph of that section:)
> If you are experiencing issues connecting to SMB shares from Pods using **hostname or FQDN**,

**Proposed Solution:**

Update [`content/en/docs/tasks/configure-pod-container/configure-gmsa.md`](https://github.com/kubernetes/website/blob/main/content/en/docs/tasks/configure-pod-container/configure-gmsa.md?plain=1):
```diff
-## Authenticating to network shares usinig hostname of FQDN
+## Authenticating to network shares using hostname or FQDN
```
(Please notice another typo: `usinig` => `using`)

---

Plus,

In L228, L248, and L260,
```diff
-```PowerShell
+```output
```

**Page to Update:**
https://kubernetes.io/docs/tasks/configure-pod-container/configure-gmsa/#:~:text=network%20shares%20usinig-,hostname%20of%20FQDN,-If%20you%20are

/language en

**Comment by jihoon-seo** [comment]
Resolved by #31865 .

**Comment by mitul3737** [comment]
Thanks for merging @jihoon-seo
